kubevirt-cdi/step2.md

```diff
@@ -0,0 +1,68 @@
+#### Introduction on Containerized Data Importer
+
+[CDI](https://github.com/kubevirt/containerized-data-importer) is an utility designed to import Virtual Machine images for use with Kubevirt.
+
+At a high level, a PersistentVolumeClaim (PVC) is created. A custom controller watches for importer specific claims, and when discovered, starts an import process to create a raw image named *disk.img* with the desired content into the associated PVC.
+
+#### Install the CDI
+
+We will first explore each component and install them. In this exercise we create a hostpath provisioner and storage class. Also we will deploy the CDI component using the Operator.
+
+`wget https://raw.githubusercontent.com/kubevirt/kubevirt.github.io/master/labs/manifests/storage-setup.yml
+kubectl create -f storage-setup.yml
+export VERSION=$(curl -s https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
+kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-operator.yaml
+kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-cr.yaml`{{execute}}
+
+Review the "cdi" pods that were added.
+
+`kubectl get pods -n cdi`{{execute}}
+
+#### Use the CDI
+
+As an example, we will import a Fedora30 Cloud Image as a PVC and launch a Virtual Machine making use of it.
+
+`kubectl create -f https://raw.githubusercontent.com/kubevirt/kubevirt.github.io/master/labs/manifests/pvc_fedora.yml`{{execute}}
+
+This will create the PVC with a proper annotation so that CDI controller detects it and launches an importer pod to gather the image specified in the *cdi.kubevirt.io/storage.import.endpoint* annotation.
+
+`kubectl get pvc fedora -o yaml
```

Comment:
Same here, 2 commands executed with one click, no time, neither explanation of what is going on, they should be separated and explained for the user to understand what is going on in the right screen.